src/app/globals.css

```diff
@@ -24,4 +24,4 @@ body {
       rgb(var(--background-end-rgb))
     )
     rgb(var(--background-start-rgb));
-}
+}
```

Comment:
To keep the consistency of project files, please change your IDE setting to insert a new-line character to the end of files.
It's the default setting in most IDE.